global.json

```diff
@@ -1,23 +1,23 @@
 {
   "tools": {
-    "dotnet": "8.0.100-preview.7.23376.3",
+    "dotnet": "8.0.100-rtm.23506.1",
```

Comment:
>    "dotnet": "8.0.100-rtm.23506.1",

@Evangelink Is this version publicly available? I don't see it on https://dotnet.microsoft.com/en-us/download/dotnet/8.0

Reply:
Nope, that just got updated via https://github.com/dotnet/roslyn-analyzers/pull/6972/commits/85b6c01db3cf4765507297b7298b8879aa52146b. @Evangelink would you mind reverting it again?

Reply:
Done